frontend/ui/data/creoptions.lua

```diff
@@ -510,6 +510,10 @@ Note that your selected font size is not affected by this setting.]]),
                     name_text = _("Max word expansion"),
                     info_text = _([[Set max word expansion as a percentage of the font size.]]),
                     event = "SetWordExpansion",
+                    other_button = { -- allow fine tuning the hidden cjk_width_scaling option (defined below)
+                        text = _("CJK scaling"),
```

Comment:
Might like a translator's note? I'm not sure how well known the term *CJK* is.

A link to https://en.wikipedia.org/wiki/CJK_characters might suffice

Reply:
Should I add it to all translated strings mentionning CJK ? Or just this one ?
There are:
```
Notification:notify(T(_("CJK width scaling set to: %1%."), value))
text = _("CJK scaling"),
name_text = _("CJK width scaling"),
info_text = _([[Increase the width of all CJK (Chinese, Japanese, Korean) chararacters by this percentage. This has the effect of adding space between these glyphs, and might make them easier to distinguish to some readers.]]),
title = _("Tailor widths and text-indent for CJK"),
```

Reply:
Indeed, never mind. I might add it to the glossary.